components/match2/commons/match_group_input_util.lua

```diff
@@ -1346,8 +1343,19 @@ function MatchGroupInputUtil.standardProcessFfaMatch(match, Parser, mapProps)
 end
 
 ---@param opponents table[]
+---@param hasNoScores boolean?
 ---@return integer[]
-function MatchGroupInputUtil.calculatePlacementOfOpponents(opponents)
+function MatchGroupInputUtil.calculatePlacementOfOpponents(opponents, hasNoScores)
+	local manualPlacements = {}
+	Array.forEach(opponents, function(opponent, opponentIndex)
+		manualPlacements[opponentIndex] = opponent.placement
+	end)
+	-- if we have no scores or a placement for each opponent return early
+	-- as there is no need to determine what is already there or can not be determined
+	if hasNoScores or Table.size(manualPlacements) == #opponents then
```

Comment:
there are cases on sc2 where no points/scores are done while at the same time not all placements are determinable
in those cases you only have manual placements while the other ones keep being empty/unset

Reply:
Can you give an example?

Reply:
https://liquipedia.net/starcraft2/Circle_Derping/Season_3_Finals

Reply:
The rest of the function should be able to handle this? (assuming that |placementX is parsed into the opponent)

Reply:
but it adds arbitrary placements instead of leaving them blank

without:
![image](https://github.com/user-attachments/assets/31595e5c-e4e4-4ffb-81d1-99828d5582bf)

with:
![image](https://github.com/user-attachments/assets/07eaf10c-a22b-4ad5-a760-d91005810678)

Reply:
Did they play the game and the placement is just not known, or what is it?

Reply:
sometimes a placement is undeterminable for some players
sometimes the placement is just unknown

Reply:
Could we use `DEFAULT_LOSS` for them? Would be most accurcate no?

Reply:
can test it in the upcoming days, but i don't see how a loss score/status will stop the placement function from applying arbitrary placements

Reply:
tested and:
- **data wise it still asigns arbitrary placement order to them**
- display wise it displays an `L` as placement in that case (which would be okay imo)
- data wise it also adjusts the (gameOpponents) status from `nil` to `'L'` (seems okay to me)

Reply:
Sounds good to me

Reply:
so we do not care about the (map) placements stored in this case?
(in arbitrary order for the ones with "missing" placement input)